tsl/test/expected/gapfill.out

```diff
@@ -144,6 +210,27 @@ FROM (VALUES (1),(2)) v(time)
 GROUP BY 1;
 ERROR:  invalid time_bucket_gapfill argument: could not infer finish boundary from WHERE clause
 \set ON_ERROR_STOP 1
+-- test time_bucket_gapfill without aggregation
```

Comment:
this behaviour is non-obvious. Why isn't this trigger gapfilling?

Reply:
because it has no aggregation